src/js/utils/linked-list.js

```diff
@@ -1,83 +1,84 @@
 export default class LinkedList {
-  constructor(options) {
```

Comment:
If called with

```js
let list = new LinkedList({
  adoptItem: () => {}
});
```

This would throw when it attempts to call `this.freeItem` and the value is `undefined`. The options object is not merged with the object passed when calling.